roles/example-cnf-app/templates/trex-app-job.yaml.j2

```diff
@@ -15,6 +15,8 @@ spec:
       - name: trex-app
         image: "{{ image_app }}"
         imagePullPolicy: "{{ image_pull_policy }}"
+        securityContext:
```

Comment:
Just curiosity, why should the client app which provides the profile to run to the trex server need to be in privileged mode?

Reply:
Hi! to be honest it is just to be able to do some manual debugging on the pod, I did not intent to merge this.